configs/unrolled_recon.yaml

```diff
@@ -90,13 +95,17 @@ simulation:
 #Training
 
 training:
-  batch_size: 16
-  epoch: 10
+  batch_size: 8
+  epoch: 50
+  #In case of instable training
+  skip_NAN: True
```

Comment:
can something like gradient clipping help for this?

Reply:
Already used but unrelated. I think i fixed the underlying issue, so those aren't really needed anymore. But this it doesn't cost much to keep the option, I left them.